static/assets.js

```diff
@@ -36,4 +36,47 @@ function hideEmptySections() {
         }
         section.style.display = 'none'
     })
-}
\ No newline at end of file
+}
+
+//  ------------    Version Filtering
+function normalize_version(raw_version) {
+    let version = raw_version
+        .replace(/^[^\d]+/, '')
+        .replace(/[^\d]+$/, '');
+    return version ? Array.from({ ...version.split('.'), length: 3 }, (v, i) => v ?? 0).join('.') : null;
+}
+
+let versionsQuery = document.querySelectorAll('.asset-card .asset-card__tags .asset-card__bevy-versions .asset-card__tag');
+let versions = [...new Set([...versionsQuery]
```

Comment:
The versions list should be sorted / it should ensure 0.12 is sorted _above_ 0.2

Reply:
I did the thing, however, a review might be needed